sql/mysqld.cc

```diff
@@ -8639,6 +8671,8 @@ SHOW_VAR status_vars[] = {
      SHOW_SCOPE_GLOBAL},
     {"Flush_commands", (char *)&refresh_version, SHOW_LONG_NOFLUSH,
      SHOW_SCOPE_GLOBAL},
+    {"Git_hash", (char *)git_hash, SHOW_CHAR},
+    {"Git_date", (char *)git_date, SHOW_CHAR},
```

Comment:
Are we missing the Rocksdb_git_hash and Rocksdb_git_date entries in SHOW_VAR? It would be fine to add them here even though they only show empty strings for now.

Reply:
This looks to be missing the SHOW_SCOPE_GLOBAL parameters.

Reply:
I can fix this. This looks to be the only issue.